src/internal/operations/restore.go

```diff
@@ -354,16 +354,17 @@ func formatDetailsForRestoration(
 	var (
 		fdsPaths = fds.Paths()
 		paths    = make([]path.Path, len(fdsPaths))
+		et       = errs.Tracker()
 	)
 
 	for i := range fdsPaths {
-		if errs.Err() != nil {
-			return nil, errs.Err()
+		if et.Err() != nil {
+			break
```

Comment:
Just to understand-  'break' is introduced to executed below block?
https://github.com/alcionai/corso/blob/88b5df728bfeda07933a936a5ab0e3485b208d6c/src/internal/operations/restore.go#L385

Reply:
Not necessarily.  It's sixes either way.